wagtailio/project_styleguide/templates/patterns/pages/feature_index_page/feature_index_page.html

```diff
@@ -0,0 +1,10 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block content %}
+    <h3>{{ page.subheading }}</h3>
```

Comment:
Note: subheading can be blank; just double-check you've added the correct handling of None.

Reply:
Thanks, handled in 99b69e6